src/components/list/EntryList.js

```diff
@@ -31,7 +31,9 @@ const EntryList = forwardRef(
       itemSize,
       itemCount,
       locations,
-      handleListEntryClick,
+      handleEntryClick,
+      handleEntryMouseEnter,
```

Comment:
Maybe we can just call these `onClick`, `onMouseEnter`, and `onMouseLeave`. No reason to reinvent the prop interface imo.

Reply:
Oh crap, yeah they should be on not handle lol.

Reply:
Do you think `Entry` is necessary? Since it's for a specific item in the list.

Reply:
Let's keep `Entry`. That's a good distinction to make.